packages/tracing/src/browser/metrics/index.ts

```diff
@@ -220,14 +215,18 @@ export function addPerformanceEntries(transaction: Transaction): void {
       _measurements[name].value = normalizedValue;
     });
 
-    if (_measurements['mark.fid'] && _measurements['fid']) {
+    const fidMark = _measurements['mark.fid'];
+    if (fidMark && _measurements['fid']) {
       // create span for FID
       _startChild(transaction, {
         description: 'first input delay',
-        endTimestamp: _measurements['mark.fid'].value + msToSec(_measurements['fid'].value),
+        endTimestamp: fidMark.value + msToSec(_measurements['fid'].value),
         op: 'web.vitals',
-        startTimestamp: _measurements['mark.fid'].value,
+        startTimestamp: fidMark.value,
       });
+
+      // Delete mark.fid as we don't want it to be part of final payload
+      delete _measurements['mark.fid'];
```

Comment:
Why do we need to delete it at the end here? Can't we just remove the line that sets `mark.fid` on `_measurements`?

Reply:
Nevermind, this is kind of a nit. It's being used to calculate the duration of the FID span but the way we're doing it now with this pattern seems a bit weird to me

Reply:
yeah it's not the cleanest, but it is the most bundle size efficient to keep it in the same object instead of constructing a new variable.

tbh it's even worth considering if the FID span is that valuable - i don't really think it is, but that's a convo for another time.